config/testing/integration-test-values.yaml

```diff
@@ -1,12 +1,48 @@
-fullnameOverride: kfp-operator-integration-tests
+namespace:
+  create: false
+fullnameOverride: kfp-operator
 containerRegistry: ""
+logging:
+  verbosity: info
 manager:
-  configuration:
-    workflowNamespace: argo
+  multiversion:
+    enabled: true
+  webhookCertificates:
+    provider: cert-manager
   argo:
     serviceAccount:
-      name: default
-      create: false
+      name: kfp-operator-argo
+      create: true
     compilerImage: kfp-operator-stub-provider
```

Comment:
I don't get why we use the stub provider image as the compiler image. If we want to stub the compiler, why don't we use a compiler stub image? Just for simplicity?